20150826 Intro to HacSoc & Linux Basics/info.yaml

```diff
@@ -0,0 +1,6 @@
+author: Thomas Murphy, Andrew Mason
+description: >
+  Andrew (@andrew_mason1) and Thomas (@thomasrussellmurphy) tell you why Linux
```

Comment:
Why do we include the twitter handles if it doesn't like to twitter? Also, that's not your twitter :stuck_out_tongue: